src/components/DpAutocomplete/DpAutocomplete.stories.mdx

```diff
@@ -2,7 +2,7 @@ import { Meta } from '@storybook/addon-docs'
 import DpAutocomplete from './DpAutocomplete'
 
 <Meta
-  title="Components/Autocomplete"
+  title="Components/DpAutocomplete"
```

Comment:
A general Question: Do we want to have the `Dp`-Prefix in the Title/Navigation or not?

Reply:
Yes we need, see in https://demosdeutschland.slack.com/archives/C5BA8ABB2/p1690353485332819?thread_ts=1690293841.102939&cid=C5BA8ABB2

When we make without `DP`-Prefix name, then the Docs will be separate from the default!

Reply:
We only need the same Title in both files I guess. It shouldn't have anything todo with the Dp if I am right.

Reply:
Yes you are right
Done in https://github.com/demos-europe/demosplan-ui/pull/396/commits/c7bab32955945945592fc6d128526305432cd559
Thanks!

Reply:
thanks for changing that!